app/src/main/java/com/release/keyneez/presentation/main/explore/ExploreFragment.kt

```diff
@@ -1,12 +1,49 @@
 package com.release.keyneez.presentation.main.explore
 
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import com.google.android.material.tabs.TabLayoutMediator
 import com.release.keyneez.R
 import com.release.keyneez.databinding.FragmentExploreBinding
+import com.release.keyneez.presentation.main.search.SearchActivity
 import com.release.keyneez.util.binding.BindingFragment
+import com.release.keyneez.util.extension.setOnSingleClickListener
 
 class ExploreFragment : BindingFragment<FragmentExploreBinding>(R.layout.fragment_explore) {
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        initExploreViewPagerAdapter()
+        initSearchClickListener()
+    }
+
+    private fun initExploreViewPagerAdapter() {
+        val viewPager = binding.vpExplore
+        val tabLayout = binding.tabExplore
+
+        val exploreTabTitles = listOf(
+            getString(R.string.explore_popular),
+            getString(R.string.explore_recent)
+        )
+
+        viewPager.adapter = ExploreAdapter(parentFragmentManager, lifecycle)
+
+        TabLayoutMediator(tabLayout, viewPager) { tab, position ->
+            tab.text = exploreTabTitles[position]
+        }.attach()
+    }
```

Comment:
```suggestion
    private fun initExploreViewPagerAdapter() {
        with (binding) {
            val exploreTabTitles = listOf(
                getString(R.string.explore_popular),
                getString(R.string.explore_recent)
            )

            vpExplore.adapter = ExploreAdapter(parentFragmentManager, lifecycle)

            TabLayoutMediator(tabExplore, vpExplore) { tab, position ->
                tab.text = exploreTabTitles[position]
            }.attach()
        }
    }
```
`with` 키워드를 활용하는 것이 더 코틀린 언어를 잘 활용한 처리라고 생각합니다!
이 기회에 scope function에 대해 공부해보셔도 좋을 것 같아요 :)